src/lando/api/tests/test_landings.py

```diff
@@ -855,27 +906,32 @@ def test_format_stack_success_changed(
     ), "Autoformat commit has incorrect commit message."
 
 
+def _scm_get_last_commit_message(scm: AbstractSCM) -> str:
+    if scm.scm_type() == SCM_TYPE_HG:
+        return HgSCM.run_hg(scm, ["log", "-r", "tip", "-T", "{desc}"]).decode("utf-8")
+    return GitSCM._git_run("log", "--pretty=%B", "HEAD^..", cwd=scm.path)
```

Comment:
Same as above, these methods should go on the `AbstractSCM` and the implementations.

Reply:
Same comment here as in `_scm_get_previous_hash` -- i.e., maybe those should be added to a mixin or test class.